infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLog.java

```diff
@@ -137,15 +142,41 @@ default OverwriteCause getOverwriteCauseForAddress(long address, LogData entry)
     /**
      * Query if the StreamLog has enough quota to accept writes
      */
-    default boolean quotaExceeded() {
-        return false;
-    }
+    boolean quotaExceeded();
 
     /**
      * Query the exact Quota value in bytes
      * @return the quota set in bytes
      */
-    default long quotaLimitInBytes() {
-        return Long.MAX_VALUE;
-    }
+    long quotaLimitInBytes();
+
+    /**
+     * Query the space used in bytes
+     * @return the space used in bytes
+     */
+    ResourceQuota getLogSizeQuota();
```

Comment:
I think we should rename this, otherwise the name per se suggests the same semantics as quotaLimitInBytes... perhaps getLogSizeUsedQuota?